src/panels/SkyBrowserPanel/BrowserTabs/Settings.tsx

```diff
@@ -1,54 +1,193 @@
     const [r, g, b] = parsedColor;
-    luaApi?.skybrowser.setBorderColor(selectedBrowserId, r, g, b);
+    luaApi?.skybrowser.setBorderColor(id, r, g, b);
   }
 
-  function deleteBrowser() {
-    if (!id) {
-      return;
-    }
-    // If there are more browsers, select another browser
-    if (browserIds.length > 1) {
-      const otherBrowsers = browserIds.filter((b) => b !== id);
-      luaApi?.skybrowser.setSelectedBrowser(otherBrowsers[0]);
-    }
-    luaApi?.skybrowser.removeTargetBrowserPair(id);
+  function setBorderRadius(newValue: number): void {
+    luaApi?.skybrowser.setBorderRadius(id, newValue);
+  }
+
+  function setVerticalFov(newValue: number): void {
+    luaApi?.skybrowser.setVerticalFov(id, newValue);
+  }
+
+  function setRightAscension(newValue: number): void {
+    luaApi?.skybrowser.setEquatorialAim(id, newValue, dec);
+  }
+
+  function setDeclination(newValue: number): void {
+    luaApi?.skybrowser.setEquatorialAim(id, ra, newValue);
   }
 
   return (
-    <Stack gap={5} my={'lg'}>
+    <Box py={'xs'}>
       <Title order={2}>Settings</Title>
-      <ColorInput
-        label={'Color'}
-        placeholder={'Set browser color...'}
-        format={'rgb'}
-        defaultValue={color}
-        onChange={setColor}
-      />
-      <Button variant={'outline'} color={'red'} onClick={deleteBrowser} mt={'lg'}>
-        Delete browser
-      </Button>
-    </Stack>
+      <Tabs defaultValue={'Settings'}>
+        <Tabs.List>
+          <Tooltip label={'Settings specific for this browser.'}>
+            <Tabs.Tab value={'Settings'}>This Browser</Tabs.Tab>
+          </Tooltip>
+          <Tooltip
+            label={`Settings for Display Copies, which are a way to render the WorldWide
+                Telescope view as a ScreenSpaceRenderable.`}
+          >
+            <Tabs.Tab value={'Display'}>Display Copies</Tabs.Tab>
+          </Tooltip>
+          <Tooltip label={`General settings for all browsers.`}>
+            <Tabs.Tab value={'General'}>General</Tabs.Tab>
+          </Tooltip>
+        </Tabs.List>
+        <Tabs.Panel value={'Settings'} py={'sm'}>
+          <NumericInput
+            value={radius}
+            label={'Border Radius'}
+            min={0}
+            max={1}
+            step={0.01}
+            clampBehavior={'strict'}
+            onEnter={setBorderRadius}
+          />
+          <ColorInput
+            label={'Color'}
+            placeholder={'Set browser color...'}
+            format={'rgb'}
+            defaultValue={color}
+            onChange={setColor}
+          />
+          <Text size={'sm'} mb={0} mt={'sm'} fw={'500'}>
+            Vertical field of view
+          </Text>
+          <Group grow>
+            <NumericPropertySlider
+              disabled={false}
+              value={fov}
+              min={0.01}
+              max={70.0}
+              step={1}
+              onInput={setVerticalFov}
+              flex={2}
+            />
+            <NumericInput
```

Comment:
I suggest you set a ` decimalScale={x}` on these inputs.  Maybe limiting it to 3-4 decimals?

Reply:
Goes for RA and Dec as well